specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-05-01/netapp.json

```diff
@@ -0,0 +1,4655 @@
+          "example": "9760acf5-4638-11e7-9bdb-020073ca7778"
+        },
+        "size": {
+          "title": "size",
+          "type": "integer",
+          "format": "int64",
+          "description": "Provisioned size of the pool (in bytes). Allowed values are in 4TiB chunks (value must be multiply of 4398046511104).",
+          "minimum": 4398046511104,
+          "maximum": 549755813888000
+        },
+        "serviceLevel": {
+          "title": "serviceLevel",
+          "type": "string",
+          "description": "The service level of the file system",
+          "enum": [
+            "Standard",
+            "Premium",
+            "Ultra"
+          ],
+          "x-ms-enum": {
+            "name": "ServiceLevel",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Standard",
+                "description": "Standard service level"
+              },
+              {
+                "value": "Premium",
+                "description": "Premium service level"
+              },
+              {
+                "value": "Ultra",
+                "description": "Ultra service level"
+              }
+            ]
+          },
+          "example": "Ultra",
+          "default": "Premium"
+        },
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure lifecycle management"
+        }
+      }
+    },
+    "capacityPoolPatch": {
+      "description": "Capacity pool patch resource",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Resource location"
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource Id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource name"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource type"
+        },
+        "tags": {
+          "description": "Resource tags",
+          "$ref": "#/definitions/resourceTags"
+        },
+        "properties": {
+          "description": "Capacity pool properties",
+          "$ref": "#/definitions/poolPatchProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "poolPatchProperties": {
+      "description": "Patchable pool properties",
+      "type": "object",
+      "properties": {
+        "size": {
+          "title": "size",
+          "type": "integer",
+          "format": "int64",
+          "description": "Provisioned size of the pool (in bytes). Allowed values are in 4TiB chunks (value must be multiply of 4398046511104).",
+          "minimum": 4398046511104,
+          "maximum": 549755813888000,
+          "default": 4398046511104
+        },
+        "serviceLevel": {
```

Comment:
There are a number of differences in this compared to 2020-06-01.  Especially this patch which is completely different in 2020-06-01.  Please diff the two api-versions and confirm that the differences are intentional.  In this case why is the patchable properties different between the two versions?

Reply:
I updated the description to reflect this.  
2020-06-01 added a qosType property to pools that turns out is not available in 2020-05. It also removed the option to update the service level of a pool in the service after it is created because now that is not allowed.